pandas/tests/window/test_ewm.py

```diff
@@ -181,3 +181,35 @@ def test_ewma_times_adjust_false_raises():
         Series(range(1)).ewm(
             0.1, adjust=False, times=date_range("2000", freq="D", periods=1)
         )
+
+
+@pytest.mark.parametrize("func", ["mean", "std", "var"])
+def test_float_dtype_ewma(func, float_dtype):
+    # GH#42452
+    expected_mean = DataFrame(
```

Comment:
Could you parameterize the expected results too like:

```
@pytest.mark.parametrize(
    "func, expected_data", 
    [["mean", [...]],
     ["std", [...],
     ...])
```

Testing on 1 column of data is fine as well e.g. `range(5)`